dsa-solutions/lc-solutions/0000-0099/0017-letter-combinations-of-a-phone-number.md

```diff
@@ -1,89 +1,104 @@
 ---
-id: letter-combinations-of-a-phone-number
+id: Letter Combinations of a Phone Number
 title: Letter Combinations of a Phone Number (LeetCode)
-sidebar_label: 0017-Letter Combinations of a Phone Number
+sidebar_label: 0017-Letter-Combinations-of-a-Phone-Number
```

Comment:
replace `sidebar_label: 0017-Letter-Combinations-of-a-Phone-Number` to `sidebar_label: 0017 Letter Combinations of a Phone Number`